### PR TITLE
/incidents/latest

### PR DESCRIPTION
**Pull request by fobabett**
gets all incidents that occured in the last 4 hours

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,5 +26,9 @@ node_modules
 
 # Users Environment Variables
 .lock-wscript
+<<<<<<< HEAD
 
 config.json
+=======
+config.json
+>>>>>>> server
```
Comment (keokilee) [inline .gitignore]: Looks like this is an unresolved merge conflict. Could you remove the green sections?

File: app/models/index.js [hunk 2]
```diff
@@ -8,10 +8,29 @@ var env       = process.env.NODE_ENV || "development";
...
+    var model = sequelize["import"](path.join(__dirname, file));
+    db[model.name] = model;
+  });
+
+Object.keys(db).forEach(function(modelName) {
+  if ("associate" in db[modelName]) {
+    db[modelName].associate(db);
+  }
+});
+
```
Comment (keokilee) [inline app/models/index.js:18]: Could you comment these things out again? Don't need them anymore.

File: logfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ERROR:  syntax error at or near "PRIVLIEGES" at character 11
+STATEMENT:  GRANT ALL PRIVLIEGES ON DATABASE hitraffic to hitraffic
+	GRANT ALL PRIVLIEGES ON DATABASE hitraffic to hitraffic
+	GRANT ALL PRIVLIEGES ON DATABASE hitraffic to hitraffic;
+ERROR:  role "hitraffic" does not exist
+STATEMENT:  GRANT ALL PRIVILEGES ON DATABASE hitraffic to hitraffic;
+ERROR:  database "hitraffic" already exists
+STATEMENT:  CREATE DATABASE hitraffic;
+LOG:  incomplete startup packet
+LOG:  incomplete startup packet
```
Comment (keokilee) [inline logfile]: Still would prefer if this were removed and added to .gitignore.

**Comment by keokilee** [comment]
Looks good. Mostly just cleanup.

**Comment by fobabett** [comment]
cleaned up

**Comment by keokilee** [comment]
Oh, looks like there's still a merge conflict. Make sure to merge from upstream/master so the button here turns green.

**Pull request merged**